components/Table/Table.vue

```diff
@@ -2,15 +2,24 @@
   <table>
     <thead>
       <tr>
-        <th v-for="(header, i) in headers" :key="i" :style="textAlign(header.align)">{{header.text}}</th>
+        <th v-for="(header, i) in headers" :key="i" :style="textAlign(header.align)">
+          <slot :name="`header_${header.key}`" :header="header">{{header.text}}</slot>
+        </th>
       </tr>
     </thead>
     <tbody>
-      <tr v-for="(item, j) in items" :key="j">
-        <td v-for="(header,i) in headers" :key="i" :style="textAlign(header.align)">
-          <slot :name="header.key" :item="item">{{ item[header.value] }}</slot>
-        </td>
-      </tr>
+      <template v-for="(item,j) in items">
+        <tr :key="item.date" @click="toggleItem(j)" @mouseover="mouseOver(j)" @mouseleave="mouseLeave(j)" :ref='"item-"+j'>
```

Comment:
Please do not use mouseover or mouseleave for that, css is perfect for that.
You can just do something like `:class="{expandable}"` and then define in the css:
```css
.expandable {
  cursor: pointer;
}

.expandable:hover {
  background-color: var(--Grey-2);
}
```

Like that, you can remove all this unnecessary complexity that will slow down the ui

Reply:
Done.